contents/articles/2016-11_multiplayer_gameplay/index.md

```diff
@@ -0,0 +1,19 @@
+--- 
+title: Exploring the Effects of Gaze Awareness on Multiplayer Gameplay
+date: Wed Nov 16 2016 17:13:32 GMT+0700 (ICT) 
+author: Pupil Dev Team 
+subtitle: "In this series of studies, the project aims to explore the invisible visual signals between players in a multiplayer gameplay..."
+featured_img: "../../../../media/images/blog/gaze-gameplayb.png"
+featured_img_thumb: "../../../../media/images/blog/gaze-gameplay.png"
+---
+
+<img src="../../../../media/images/blog/gaze-gameplay.png" style="width: 70%; margin: auto;" class='Feature-image u-padBottom--1' alt="The Van Gogh Museum Eye-tracking Project">
+Image Source: [Exploring the Effects of Gaze Awareness on Multiplayer Gameplay PDF](http://www.socialnui.unimelb.edu.au/publications/2016-SocialNUI-Newn-3.pdf)
+
+In this series of studies, the project aims to explore the invisible visual signals between players in a multiplayer gameplay! The gameplay would simulate different gaze conditions to test players interactions, varying in levels of gaze information from invisible to visible. 
+
+Gaze can provide visual information of the player's intention. During gameplay, players monitor each other's interactions to gain and evaluate opponents intentions and formulate strategies based upon the visual information.
+
+In conventional social interactions, gaze is largely invisible. But in a competitive gameplay environment, a player's interactions can reveal valuable information for other players. 
+
```

Comment:
Pupil hardware and software was used in the study to track the gaze of two players during co-located gameplay. Gaze information is processed and visualized on top of the table top board game via a projector - "augmenting both the board and the game objects."